spyder/widgets/fileswitcher.py

```diff
@@ -663,10 +663,12 @@ def setup_file_list(self, filter_text, current_path):
                     text_item += " [{0:} {1:}]".format(self.line_count[index],
                                                        _("lines"))
                 if max_width > self.list.width():
-                    text_item += u"<br><i style='color:{0:}'>{1:}</i>".format(
+                    text_item += (u" <i style='color:rgb(153,153,153)'>"
```

Comment:
@juanis2112 please put the `rgb(153,153,153)` in a constant, something like `PATH_FG_COLOR` below [`MAX_WIDTH`](https://github.com/spyder-ide/spyder/blob/4db4465f09c8d48cc26dc7feed539db532ea4677/spyder/widgets/fileswitcher.py#L231) and use it as the first argument (`{0:}`) of the format (replacing the `ima.MAIN_FG_COLOR`). Also I think that we should remove the italics (maybe we can use `p` instead to wrap the path) You should get something similar to this:

```python
text_item += u"<br><p style='color:{0:}'>{1:}</p>".format(
    self.PATH_FG_COLOR, short_paths[index])
```

Reply:
Is ``#999999`` a standard color already in use in our dark theme? If not, can we pick one that is, so our color scheme is more consistant and we don't have a dozen different shades of gray floating around our codebase?